src/test/scala/com/gu/acquisition/services/DefaultAcquisitionServiceSpec.scala

```diff
@@ -15,7 +15,7 @@ class DefaultAcquisitionServiceSpec extends AsyncWordSpecLike with Matchers with
 
   val submission = AcquisitionSubmission(
     OphanIds(None, None, Some("None")),
-    GAData("support.code.dev-theguardian.com", None, None),
+    GAData("support.code.dev-theguardian.com", "GA1.1.1633795050.1537436107", None, None),
```

Comment:
Is this hardcoded id require?

Reply:
Since you are using it in two places is it possible to put it in share config file?

Reply:
That's a test

Reply:
Sorry, my bad, I saw it first in https://github.com/guardian/acquisition-event-producer/pull/32/files#diff-0e7e3ecd08f911e302321bd698ff8a26R22 without realising that was inside a `/test/` folder.